### PR TITLE
xk6-enhanced extension renamed to xk6-ts

### PR DESCRIPTION
**Pull request by szkiba**
## What?

xk6-enhanced extension renamed to xk6-ts

## Checklist

- [x] I have used a meaningful title for the PR.
- [x] I have described the changes I've made in the "What?" section above.
- [x] I have performed a self-review of my changes.
- [ ] I have run the `npm start` command locally and verified that the changes look good.

- [ ] I have made my changes in the `docs/sources/next` folder of the documentation.
- [ ] I have reflected my changes in the `docs/sources/v{most_recent_release}` folder of the documentation.
- [ ] I have reflected my changes in the relevant folders of the two previous k6 versions of the documentation (if still applicable to previous versions).

- [ ] I have made my changes in the `docs/sources/next` folder of the documentation.

### CODE REVIEW

**Comment by heitortsergent** [comment]
@szkiba can you update the pages for the Grafana docs as well? It would be these two:

- https://github.com/grafana/k6-docs/blob/main/docs/sources/next/extensions/explore.md?plain=1#L99
- https://github.com/grafana/k6-docs/blob/main/docs/sources/v0.50.x/extensions/explore.md?plain=1#L98

I can do it as well and update the PR if you want, just let me know! 🙇

File: src/data/doc-extensions/extensions.json [hunk 1]
```diff
@@ -947,9 +947,9 @@
       "cloudEnabled": false
     },
     {
-      "name": "xk6-enhanced",
-      "description": "Add enhanced JavaScript compatibility (TypeScript, import JSON files, etc) to k6",
-      "url": "https://github.com/szkiba/xk6-enhanced",
+      "name": "xk6-ts",
+      "description": "TypeScript support for k6",
```
Comment (heitortsergent) [inline src/data/doc-extensions/extensions.json:951]:
```suggestion
      "description": "Add TypeScript support for k6",
```

**Comment by szkiba** [comment]
> @szkiba can you update the pages for the Grafana docs as well? It would be these two:
> 
> - https://github.com/grafana/k6-docs/blob/main/docs/sources/next/extensions/explore.md?plain=1#L99
> - https://github.com/grafana/k6-docs/blob/main/docs/sources/v0.50.x/extensions/explore.md?plain=1#L98
> 
> I can do it as well and update the PR if you want, just let me know! 🙇 


I would appreciate if you can do it...

**Comment by heitortsergent** [comment]
@szkiba actually, it looks like you opened the PR from your repository, and not from a branch here. :( Could you give me write access to your repo, or reopen the PR from a branch?

**Comment by szkiba** [comment]
> [heitortsergent](/heitortsergent)

I added you as contributor.

**Review by heitortsergent — APPROVED** [review]

**Comment by szkiba** [comment]
Thank you @heitortsergent

**Pull request merged**